features/checkout/paying_for_order/preventing_to_change_payment_method_of_already_cancelled_order.feature

```diff
@@ -0,0 +1,31 @@
+@paying_for_order
+Feature: Preventing to change payment method of the cancelled order
+    In order to not be able to change payment method of the order that is already cancelled
+    As a Customer
+    I want to be prevented from changing payment method of the cancelled order
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And the store has a product "Angel T-Shirt"
+        And the store ships everywhere for Free
+        And the store allows paying with "Cash on Delivery"
+        And the store also allows paying with "Bank Transfer"
+        And I am a logged in customer
+        And I placed an order "#00000666"
+        And I bought a single "Angel T-Shirt"
+        And I addressed it to "Lucifer Morningstar", "Seaside Fwy", "90802" "Los Angeles" in the "United States" with identical billing address
+        And I chose "Free" shipping method with "Cash on Delivery" payment
+
+    @api @no-ui
+    Scenario: Being prevented to change payment method of cancelled order
+        Given this order was cancelled
+        When I change my payment method to "Bank Transfer"
+        Then I should be notified that I can no longer change payment method of this order
+
+    @ui @no-api
+    Scenario: Being prevented to change payment method of cancelled order
+        When I browse my orders
+        And I am on the change payment method page
+        Then this order was cancelled
+        And I switch to "Bank Transfer" payment method
```

Comment:
there is some problem with the structure given ->when -> then , because step `And I switch to "Bank Transfer" payment method` looks like `when`,  `Whens` are the `act` and `thens` are `assert`,
WDYT about:
```suggestion
        Given I am on the change payment method page
        And this order was cancelled
        When I switch to "Bank Transfer" payment method
        Then I should be notified that I can no longer change payment method of this order
```